lib/python/rose/apps/rose_bunch.py

```diff
@@ -342,11 +342,16 @@ def connect(self):
     def create_tables(self):
         """Create tables as appropriate"""
         existing = []
+        first_run = os.environ["CYLC_TASK_SUBMIT_NUMBER"] == "1"
```

Comment:
@matthewrmshin - yeah I appreciate that but this is the best catch for it I can think of. Any better/other suggestions?